designer/client/component-condition-create.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ComponentTypeEdit from './component-type-edit'
-import ConditionalComponentTypes from 'digital-form-builder-engine/src/conditional-component-types'
+import ConditionalComponentTypes from 'digital-form-builder-model/lib/conditional-component-types'
```

Comment:
sorry I didn't catch this before..! take a look at `-model/src/lib`. I'm exporting everything that's meant to be consumed in the package. Then in in `model/index`, I'm importing (the transpiled) lib.js. This way you don't have to do (or shouldn't?) import `'digital-form-builder-model/lib`. you can just do `import ... from digital-form-builder-model`. This is more in line with how npm packages are consumed

Reply:
Fixed